http/src/request/user/get_current_user_guilds.rs

```diff
@@ -1,9 +1,11 @@
 use crate::request::prelude::*;
+use serde::{Deserialize, Serialize};
 use std::{
     error::Error,
     fmt::{Display, Formatter, Result as FmtResult},
 };
-use twilight_model::{guild::PartialGuild, id::GuildId};
+use twilight_model::guild::Permissions;
+use twilight_model::id::GuildId;
```

Comment:
```suggestion
use twilight_model::{guild::Permissions, id::GuildId};
```
This should stay consistent.

Reply:
Ah yah whoops, I guess rust analyzer didn't do it for me. I'll fix that